src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/components/InvestActions.vue

```diff
@@ -165,6 +167,7 @@ async function handleTransaction(tx): Promise<void> {
         Goals.LiquidityAdded,
         bnum(fiatTotal.value).times(100).toNumber() || 0
       );
+      await refetchBalances.value();
```

Comment:
It's not as relevant because the user would have likely performed the approval before the final transaction. So the blocknative event would have been triggered twice, if the first event didn't capture the approval update the second one would. That would sync the approval state before they are likely to enter another flow where approvals are required.